### ISSUE TITLE
Toggle shortcut doesn't work on macOS Catalina

### ISSUE THREAD

**Issue by arabello** [federico-terzi/espanso#108]
Hi @federico-terzi ,
the toggle shortcut mentioned [here](https://espanso.org/docs/get-started/#useful-shortcuts) seems to not work on my system (Catalina 10.15).

Tell me if you need specific tests I can do to find out the problem.

**Comment by federico-terzi** [comment]
Hi @arabello,
Thank you very much for notifying me, I'll look into the problem as soon as possible.

**Comment by federico-terzi** [comment]
I tested espanso on macOS Catalina and unfortunately I couldn't reproduce the issue.
What kind of behavior are you experiencing?

If you press the shortcut, does it ignore it and keep working instead?

Thanks for your help

**Comment by arabello** [comment]
Hi,
I just found out what was my problem. I'm using an external Windows keyboard, so that I changed the modifier keys resulting in a broken toggle shortcut.

 To recap, _using an external USB keyboard_, all works fine with the following settings:
![Screenshot 2019-10-29 at 22 46 52](https://user-images.githubusercontent.com/16003164/67812381-5ee01a80-fa9f-11e9-9cfc-5613e87c227b.png)

Instead, the shortcut doesn't work if the `option` key is switched with `command` such as follow:
![Screenshot 2019-10-29 at 22 47 29](https://user-images.githubusercontent.com/16003164/67812568-ca29ec80-fa9f-11e9-80c6-d7fa52081400.png)

I'm aware that this is no longer a bug, but only a specific issue case, so I close the issue.
